### DISCUSSION TITLE
v1.0.0 🎊 Release

### DISCUSSION THREAD

**Discussion by mulaRahul** [mulaRahul/keyviz#8] [Announcements]
## ✨ Major Changes
- Settings window UI re-design 6ac75e2b166030a557c987daf3fb12b33136deab
- Removed "flat" style keycaps 9dc6f032abf41dba6b0093062ffdf8cfc4ebbb63

(_the above changes were made due to a conflict with another application_)

## 🆕 New Features
- "**Revert**" settings option added to general settings 130c7cf3b54dcebebed1ffc9e8a45eff2a16f9e0 upon #5 
- "**Exit Without Saving**" option added to the exit dialog of settings window 28da1eb611334eb8883f0e7aaa2c6bd4c2e43e3a upon #5 
- Exposed the "**Animation Speed**" setting 5363983225af030fb017278113842441786197b2 upon #4

## 🐛 Bug Fixes
- #2 use `WindowEffect.transparent` as window effect cb2fd4d5a5ce35208e5579328c454a0182091d75

<hr /><em>This discussion was created from the release <a href='https://github.com/mulaRahul/keyviz/releases/tag/v1.0.0'>v1.0.0 🎊 Release</a>.</em>

**Comment by Nitti-Nit** [comment]
It would be great to have an option allowing for ignoring some keys. Great app
